ILRepack.IntegrationTests/ILRepack.IntegrationTests.csproj

```diff
@@ -30,6 +30,7 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="FSharp.Core, Version=4.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
```

Comment:
It might come from [SourceLink.Core](https://www.nuget.org/packages/SourceLink.Core) who states FSharp.Core dependency.